release.md

```diff
@@ -48,7 +112,7 @@ The *driver version number* represents the build version of kernel-space drivers
 
 > _Note that `API_VERSION` and `SCHEMA_VERSION` are only used internally. On the other hand, only the **driver version number** will be used **to tag a new release**._
```

Comment:
For (newer) devs a bit more clarity on why both `API_VERSION` and `SCHEMA_VERSION` are needed and what the exact distinction is would be nice (if possible), https://github.com/falcosecurity/libs/blob/master/driver/README.VERSION.md. Schema version is more intuitive to me, but for some PRs I wouldn't be too sure what has to be bumped how.